LayoutTests/fast/webgpu/nocrash/RenderBundle_WRITE.html

```diff
@@ -1,6 +1,6 @@
 <!-- webkit-test-runner [ enableMetalShaderValidation=true ] -->
 <script>
-  globalThis.testRunner?.waitUntilDone();
+  globalThis.testRunner?.dumpAsText(); globalThis.testRunner?.waitUntilDone();
```

Comment:
I don't think we should be lazy in this either.
E.g it's not typical to have  a();b(); type code, rather this code should be business as usual.
If you did search-and-replace in your editor, typically option-enter or something like that would insert a line feed into the replace box, and we can have normal code.

Reply:
sure I can add a newline